src/index.js

```diff
@@ -9,5 +9,58 @@ import './css/base.scss';
 
 // An example of how you tell webpack to use an image (also need to link to it in the index.html)
 import './images/turing-logo.png'
+import Clue from '../src/Clue'
+import '../src/Game'
+import '../src/Player'
+import '../src/Round'
+// import { promises } from 'dns';
```

Comment:
I'm going to assume that this appeared through accidental autocomplete -- DNS is a node module that I don't have much experience with, and you shouldn't need it. Unless @hannahhch has any more context on this?

Reply:
Yeah it looks like a case of your text editor trying to be helpful. You shouldn't need to worry about dns!